crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -745,7 +745,15 @@ define_tedge_config! {
 
             /// The maximum number of software packages reported for each type of software package
             #[tedge_config(example = "1000", default(value = 1000u32))]
-            max_packages: u32
+            max_packages: u32,
+
+            /// The filtering criterion, in form of regex, that is used to filter packages list output
+            #[tedge_config(example = "tedge.*")]
+            include: String,
+
+            /// The filtering criterion, in form of regex, that is used to filter out packages from the output list
+            #[tedge_config(example = "tedge.*")]
+            exclude: String,
```

Comment:
suggestion: for examples add packages mentioned in the issue, i.e. `libc.*, iptables-module-.*, kernel-.*`

Reply:
Yes, example makes sense...I'm almost tempted to set the default to the following, as this would mean out of the box thin-edge.io will send back something sensible (reducing the network bandwidth)

```sh
^(glibc|libc|kernel-|iptables-module|).*
```

@didier-wenzek Or would this default "do too much", and be unexpected for users?

Reply:
As an example, this makes sense but not as the default. It's better to let the user hide packages himself.

Reply:
ok, agreed. let's leave the default empty. Though adding the following regex in the --exclude example will be helpful.

```
^(glibc|libc|kernel-|iptables-module).*
```